### PR TITLE
add possibility to set specific results and fix a bug in algorithm

### LINKED ISSUE #71: Add possibility to set specific results for some students

### LINKED ISSUE #83: backend: still a bug when not all students have voted

### PR DESCRIPTION
**Pull request by Lordnoname**
Closes #83 
Closes #71

### CODE REVIEW

File: frontend/src/components/listVotings/voting-details-admin.component.html [hunk 1]
```diff
@@ -40,18 +40,48 @@ <h4 mat-line>{{ user.firstname }} {{ user.lastname }}</h4>
...
-          <mat-option *ngFor="let user of voting.voted" [value]="user.username">
-            <span>{{ user.firstName }} {{ user.lastName }}</span> |
-            <small>Klasse: {{ user.form }}</small>
-          </mat-option>
-        </mat-autocomplete>
-      </mat-form-field>
+      <h4>Erstwahl von Schüler priorisieren</h4>
+      <p *ngIf="!voting.votes || voting.votes.length === 0">Es sind keine Schülerwahlen vorhanden. Bitte fügen Sie zuerst Wahlen hinzu, um den Schülern eine priorisierte Erstwahl zu ermöglichen.</p>
+      <div *ngIf="voting.votes && voting.votes.length !== 0">
+        <mat-form-field>
```
Comment (gilgda) [inline frontend/src/components/listVotings/voting-details-admin.component.html]: A margin-left would be nice to have a better view to the dialogs

Comment (Lordnoname) [reply frontend/src/components/listVotings/voting-details-admin.component.html]: Setting a default value for `prioritized` would be to much effort. Currently the default value is `null`.

Comment (gilgda) [reply frontend/src/components/listVotings/voting-details-admin.component.html]: Ok, but there is an error occurring when users vote as there seems to be no default value. Did you already test this on your system?
vote bug

Comment (Lordnoname) [reply frontend/src/components/listVotings/voting-details-admin.component.html]: Okay....did not know about the sql syntax....thank you for resolve that :)

**Comment by Lordnoname** [comment]
@gilgda You need to renew your database setup due to the added column :)

**Comment by gilgda** [comment]
Oh thanks here 😅 sorry then it looks fine to me

**Pull request merged**